community/openebs/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+version: 0.7.0
+name: openebs
+appVersion: 0.7.0
+tillerVersion: ">=2.8.0"

```

Comment:
I've demoed the same version, but it turns out that I was missing the updated index.yaml file in the PR. 
After creating a new index.yaml now the chart is visible in the catalog. Pushing the changes soon.

Reply:
Thanks @muratkars I assume you mean Chart.yaml, right? index.yaml should not be in any PRs, as it is updated by our build process.

Reply:
We started supporting since 2.6.0, but we have been using 2.7.2 for some time. So i've changed it to 2.7.2 and tested successfully on ICP 2.1.0.2 and 2.1.0.3.